WooCommerce/WooCommerceTests/ViewRelated/Products/Filters/FilterProductListViewModelTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+@testable import WooCommerce
+
+final class FilterProductListViewModelTests: XCTestCase {
+    func testCriteriaWithDefaultFilters() {
+        let filters = FilterProductListViewModel.Filters()
+        let viewModel = FilterProductListViewModel(filters: filters)
+        let expectedCriteria = FilterProductListViewModel.Filters(stockStatus: nil, productStatus: nil, productType: nil, numberOfActiveFilters: 0)
+        XCTAssertEqual(viewModel.criteria, expectedCriteria)
+    }
```

Comment:
Even if we don't use the Given-When-Then comments, I think it's still a good idea to space these out so we can easily identify which are the actions and assertions. 

```suggestion
    func testCriteriaWithDefaultFilters() {
        let filters = FilterProductListViewModel.Filters()

        let viewModel = FilterProductListViewModel(filters: filters)

        let expectedCriteria = FilterProductListViewModel.Filters(stockStatus: nil, productStatus: nil, productType: nil, numberOfActiveFilters: 0)
        XCTAssertEqual(viewModel.criteria, expectedCriteria)
    }
```

Reply:
it's better to add the Given-When-Then comments! updated here: https://github.com/woocommerce/woocommerce-ios/pull/2328/commits/8a103d5262069c107c87077d4246d17f4f63bdbd